src/handler.rs

```diff
@@ -27,9 +63,20 @@ impl EventHandler for Handler {
 
 		component.create_response(&ctx, response).await.unwrap();
 	}
+}
+
+#[serenity::async_trait]
+impl EventHandler for Handler {
+	async fn interaction_create(&self, ctx: Context, interaction: Interaction) {
+		match interaction {
+			Interaction::Command(command) => self.command_create(ctx, command).await,
+			Interaction::Component(component) => self.component_create(ctx, component).await,
+			_ => (),
+		}
+	}
 
 	async fn message(&self, ctx: Context, message: Message) {
-		if message.author.bot {
+		if message.author.id == ctx.cache.current_user().id {
```

Comment:
This change is not in the scope of this pull request neither mentioned in the description.